### ISSUE TITLE
AddPed() in gamemode script can cause crashes and deformed ped models

### ISSUE THREAD

**Issue by LemonHaze420** [gtaco-op/vc-coop#12]
Can be reproduced with the following gamemode lua script:-
```lua
function onServerStart()
	print("Default Gamemode Started")
	
	AddVehicle(130, 530.393555, 618.161865, 10.901563)
	AddVehicle(131, 538.117798, 626.908203, 10.901563)
	
	AddPed(1, 532.501892, 636.292664, 11.043379, wander)
end
```

**Comment by LemonHaze420** [comment]
Temporary fix in commit: a595b57eced98076ff0e2d1ebd7a6c401b572113

Issue is in relation to ped data not being initialized correctly server-side.

**Comment by LemonHaze420** [comment]
Fixed in latest revision.
